src/app/services/redux-constants.ts

```diff
@@ -20,3 +20,6 @@ export const SAMPLES_FETCH_SUCCESS = 'SAMPLES_FETCH_SUCCESS';
 export const SAMPLES_FETCH_ERROR = 'SAMPLES_FETCH_ERROR';
 export const SAMPLES_FETCH_PENDING = 'SAMPLES_FETCH_PENDING';
 export const CHANGE_THEME_SUCCESS = 'CHANGE_THEME_SUCCESS';
+export const ADAPTIVE_FETCH_SUCCESS = 'ADAPTIVE_FETCH_SUCCESS';
```

Comment:
Action constants sound better when you start with the verb ie `FETCH_ADAPTIVE_CARD_SUCCESS` :)